acropolis-shell/src/main/kotlin/org/ephyra/acropolis/shell/CreateCommand.kt

```diff
@@ -40,8 +40,7 @@ class CreateCommand {
         val project = appState.currentProject
         if (project != null) {
             systemSoftwareService.create(project.id, name)
-        }
-        else {
+        } else {
```

Comment:
yuck

Reply:
is that the default formatter? you haven't modified your rules?

If it is the default then let's just go with it

Reply:
Tis default indeed. And my favourite <3